dist/common/scripts/scylla_util.py

```diff
@@ -299,19 +300,24 @@ def is_recommended_instance(self):
         if self.is_recommended_instance_size() and not self.is_unsupported_instance_class() and self.is_supported_instance_class():
             # at least 1:2GB cpu:ram ratio , GCP is at 1:4, so this should be fine
             if self.cpu/self.memoryGB < 0.5:
-              # 30:1 Disk/RAM ratio must be kept at least(AWS), we relax this a little bit
-              # on GCP we are OK with 50:1 , n1-standard-2 can cope with 1 disk, not more
               diskCount = self.nvmeDiskCount
               # to reach max performance for > 16 disks we mandate 32 or more vcpus
               # https://cloud.google.com/compute/docs/disks/local-ssd#performance
               if diskCount >= 16 and self.cpu < 32:
+                  logging.warning("This machine doesn't have enough CPUs for allocated number of NVMEs (at least 32 cpus for >=16 disks). Performance will suffer.")
                   return False
               diskSize= self.firstNvmeSize
               if diskCount < 1:
                   return False
+              max_disktoramratio = 60
+              # 30:1 Disk/RAM ratio must be kept at least(AWS), we relax this a little bit
+              # on GCP we are OK with {max_disktoramratio}:1 , n1-standard-2 can cope with 1 disk, not more
               disktoramratio = (diskCount*diskSize)/self.memoryGB
-              if (disktoramratio <= 50) and (disktoramratio > 0):
+              if (disktoramratio <= max_disktoramratio) and (disktoramratio > 0):
+                  logging.warning(f"Instance disk-to-RAM ratio is %s, which is higher than the recommended ratio %s. Performance may suffer.",disktoramratio,max_disktoramratio)
```

Comment:
The warning should not be here, right?

Reply:
Please use f-strings.

Reply:
it should
it warns when this setup isn't optimal

Reply:
(note I didn't use your logic from above
I think this should still fail and force iotune to be run)

Reply:
(and these two patches should be squashed to one)

Reply:
I don't understand.

Let's assume that `disktoramratio` is 49, for example, which is fine. The `if` statement expression will evaluate to `true`:

```python
if (disktoramratio <= max_disktoramratio) and (disktoramratio > 0):
# =>
if (49 <= 60) and (49 > 0):
```

We will then log a warning that says

```
Instance disk-to-RAM ratio is 49, which is higher than the recommended ratio 60. Performance may suffer.
```

Why is this what we want?

(Also, please use f-strings: https://realpython.com/python-f-strings/)

Reply:
Yeah, please squash the commits, it's easier to review it that way.

Reply:
ah, yes, sorry, will rephrase the log and fix it appropriately